src/templates/genes.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'base.html' %}
+{% set active_page = "genes" %}
+
+{% block content %}
+
+<!-- Begin Page Content -->
+<div class="container-fluid">
+
+    <!-- Page Heading -->
+    <div class="d-sm-flex align-items-center justify-content-between mb-4">
+        <h1 class="h3 mb-0 text-gray-800">Genes</h1>
+        <small>gene</small>
+    </div>
+
+    <!-- Content Row -->
+    <div class="row">
+        <div class="col">
+            <div class="card shadow mb-4">
+                <div class="card-body">
+                    <div class="table-responsive">
+                        <table class="table table-striped table-hover" id="genes">
+                            <thead>
+                                <tr>
+                                    <th>#</th>
+                                    <th>gene</th>
+                                    <th>Total variants</th>
+                                    <!--<th>Total patients</th>-->
+                                </tr>
+                            </thead>
+                            <tbody>
+                                {# data will be provided with api #}
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+</div>
+<!-- /.container-fluid -->
+
+{% endblock %}
```

Comment:
I see a warning: "No newline at end of file".

Reply:
not needed for HTML